src/main/java/com/celements/search/web/classes/WebSearchClassPackage.java

```diff
@@ -12,7 +12,7 @@
 @Component(WebSearchClassPackage.NAME)
 public class WebSearchClassPackage extends AbstractClassPackage {
 
-  public static final String NAME = "webSearch";
+  public static final String NAME = "websearch";
```

Comment:
I think, the idea is, that all XWikiPreferences with a configured "webSearch" should automatically use the new definition instead of requiring the configuration to change to switch.